examples/src/main/scala/org/apache/spark/examples/HdfsTest.scala

```diff
@@ -41,6 +41,8 @@ object HdfsTest {
       val end = System.currentTimeMillis()
       println(s"Iteration $iter took ${end-start} ms")
     }
+    println(s"File contents: ${file.map(_.toString).take(1).mkString(",").slice(0, 10)}")
+    println(s"Returned length(s) of: ${file.map(_.length).collect().mkString(",")}")
```

Comment:
This is printing the same thing as before my previous comment, right? In my run this printed pages and pages of "1,1,1,1,1,1,1,1,1" which is not really useful even for debugging things.